lib/pbench/agent/tool_data_sink.py

```diff
@@ -497,7 +502,10 @@ def _wait_for_all_data(self):
         Waiting is a no-op for all states except 'send'.  In the 'send' state,
         we are expecting to hear from all registered tool meisters.
         """
-        assert self.state == "send", f"expected state 'send' not '{self.state}'"
+        assert self.state in (
+            "send",
+            "sysinfo",
```

Comment:
I see at least three instances of the ("send", "sysinfo") tuple; I wonder if a class `send_data_states` property might not be a bad idea, to encapsulate this? Or even an `is_expecting_data()` method?

Reply:
Fixed.